examples/otel-logs-splunk/README.md

```diff
@@ -14,3 +14,35 @@ $> docker-compose up
 Splunk will become available on port 18000. You can login on [http://localhost:18000](http://localhost:18000) with `admin` and `changeme`.
 
 Once logged in, visit the [search application](http://localhost:18000/en-US/app/search) to see the logs collected by Splunk.
+
+If you would like to see the metrics exposed by the OpenTelemetry Collector, 
+you can visit [http://localhost:8888](http://localhost:8888) to see the metrics emitted by the collector.
+
```

Comment:
This sentence reads redundant - I'd go for "Visit [http://localhost:8888](http://localhost:8888) to see the metrics emitted by the OpenTelemetry Collector."

Reply:
Addressed, thanks!